test/modules/packages/Crypto/saru/hashing/hash2.bad

```diff
@@ -0,0 +1 @@
+error: halt reached - A digest with the name 'SHA1024' doesn't exist.
```

Comment:
.bad files are meant to go with tests that have a .future file.  If the .bad file contains the expected output it should be named hash2.good.  If you consider this to be incorrect output you should add a .future file and .good file as described [here](https://github.com/chapel-lang/chapel/blob/master/doc/rst/developer/bestPractices/TestSystem.rst)